sunpy/net/jsoc/jsoc.py

```diff
@@ -646,6 +647,7 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
                 print_message = "{0} URLs found for download. Full request totalling {1}MB"
                 print(print_message.format(len(urls), request._d['size']))
             for aurl, fname in zip(urls, paths):
+                log.debug(f'Adding {aurl} to download queue.')
```

Comment:
Well and truly baffled, this doesn't work (ie. a log message doesn't get printed), but if I put `print(log)` before this line it does...

Reply:
Yeah this looks like it should be fine.

Reply:
> Well and truly baffled, this doesn't work (ie. a log message doesn't get printed), but if I put `print(log)` before this line it does...

Er, are you turning on debug output properly?
```python
log.setLevel("DEBUG")
```
I just tried this on my machine, and the debug log message shows up fine.

Reply:
```
DEBUG: Adding http://jsoc.stanford.edu/SUM14/D1333490357/S00000/hmi.m_45s.20200101_010130_TAI.2.magnetogram.fits to down
load queue. [sunpy.net.jsoc.jsoc]
```

Reply:
Yeah, I'm setting the level properly. Maybe it's just something funny with my setup.